handlers/bootstrap.go

```diff
@@ -40,6 +40,8 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) error
 		return publishReply(reply, apiClient)
 	}
 
+	publishTransitioningReply("Installing Rancher agent...", event, apiClient)
```

Comment:
Only reason I did that was because Docker messages also displays to maintain some type of consistency.